Documentation/Extensions/HowToFindExtensions.rst

```diff
@@ -0,0 +1,75 @@
+.. include:: /Includes.rst.txt
+
+.. _how-to-find-extensions:
+
+======================
+How to find extensions
+======================
+
+For beginners, finding the right extensions can enhance their TYPO3 experience
+significantly. Here is a brief guide on how to search for extensions and where
+to find the necessary information.
+
+#.  `TYPO3 Extension Repository (TER) <https://extensions.typo3.org/>`__
+
+    The TYPO3 Extension Repository (TER) is online platform, that hosts
+    thousands of extensions created by the TYPO3 Community.
+    Here you can:
+
+    *   Search by keyword
+    *   Explore categories
+    *   Filter results
+
+#.  `Packagist <https://packagist.org//>`__
+
+    Packagist serves as the default package repository for PHP packages using Composer,
+    including TYPO3 extensions. You can search for TYPO3 extensions on Packagist
+    by entering relevant keywords. Many TYPO3 extensions are available via Composer,
+    which allows for easier dependency management and installation. This is
+    particularly useful for developers looking to integrate extensions into
+    their projects.
```

Comment:
You can also filter by TYPO3 extensions:

![image](https://github.com/user-attachments/assets/348ef42f-af47-413d-a201-5e269b9a8fd8)

https://packagist.org/?type=typo3-cms-extension